Khan/wonder-blocks
# Use wonder blocks Icon instead of img in Banner

## Summary:
- Use Wonder Blocks Icon for the icons in Banner.
- Remove the use of `img` for icons as well as the svg files.
- Add aria-label to icons
- Add tests for kind

This PR should get rid of the code coverage complaints.

Updates to come in future PRs:
- Changing the buttons to be small (not adding this in the current PR so that I can keep my branches consistent)
- Comprehensive accessibility features and testing
  - This includes role and aria-live

**Storybook publish:**
https://5e1bf4b385e3fb0020b7073c-opbliurcxg.chromatic.com/

Issue: https://khanacademy.atlassian.net/browse/WB-1385

### A11y Demo
https://user-images.githubusercontent.com/13231763/197080461-752b974f-09bc-4848-92b8-68bd76d2fc03.mov


## Test plan:
- Go to http://localhost:6061/?path=/story/banner--kinds
- Use a screenreader and confirm the icons are labeled correctly.

`yarn jest packages/wonder-blocks-banner/src/components/__tests__/banner.test.js`
